third_party/nvfuser/csrc/index_compute.cpp

```diff
@@ -1239,6 +1240,10 @@ indexMapFromTV(
       idx = loop->start();
     }
 
+    if (rotated_loop.count(loop) > 0 && zero_loops.count(loop) == 0) {
```

Comment:
`rotated_loop` depend on the expr you are visiting. For example, if you have
```
statement1(0)
statement2(0)
for i ...
  statement3(i)
  statement4(i)
  if LoopRotation:
    statement1(i+1)
    statement2(i+1)
```
Then `for i...` is a rotated loop only if you are visiting the `statement1` and `statement2` inside the `if LoopRotation`, otherwise, for example if you are visiting `statement3` or `statement4`, then this `rotated_loop` will be empty

Reply:
OK, makes sense.